src/methods/function.js

```diff
@@ -584,6 +585,7 @@ export function FunctionInitialize(
   F.$FormalParameters = ParameterList;
 
   // 7. Set the [[ECMAScriptCode]] internal slot of F to Body.
+  (Body: any).uniqueTag = functionBodyUniqueTagSeed++;
```

Comment:
We also avoid using any whenever possible. In this case, the unique tag is not necessary as the object identity can be used instead.

Reply:
Will add comment to explain the reason.